kudu/swagger.json

```diff
@@ -4368,6 +4368,12 @@
                             "type": "object"
                         }
                     },
+                    "201": {
+                        "description": "OK",
```

Comment:
What's the description used for?  Obviously 201 is not simply OK.

Reply:
The description is ignored. But I can fix them anyways https://github.com/EricJizbaMSFT/kudu/commit/28da8eba09ba2d11ece514d570737e3ab18e12e5